pkg/secrets/passphrase/manager.go

```diff
@@ -225,7 +225,12 @@ func PromptForNewPassphrase(rotate bool) (string, secrets.Manager, error) {
 			firstMessage = "Enter your new passphrase to protect config/secrets"
 
 			if !isInteractive() {
-				return "", nil, fmt.Errorf("passphrase rotation requires an interactive terminal")
+				text, err := io.ReadAll(os.Stdin)
```

Comment:
Perhaps this should read just the next line from stdin?
https://pkg.go.dev/bufio#example-Scanner-Lines